src/invert.ts

```diff
@@ -0,0 +1,35 @@
+import { purry } from './purry';
+
+type Invertable = { [index: string]: string } | { [index: number]: string };
+type Inverted = { [index: string]: string };
+
+/**
+ * Returns an object whose keys are values are swapped. If the object contains duplicate values,
+ * subsequent values will overwrite previous values.
+ * @param object the object
+ * @signature
+ *    R.invert(object)
+ * @example
+ *    R.invert({ a: "d", b: "e", c: "f" }) // => { d: "a", e: "b", f: "c" }
+ *    R.invert({}) // => {}
+ *    R.pipe(
+ *      { a: "d", b: "e", c: "f" }
+ *      R.invert()
+ *    ); // => { d: "a", e: "b", f: "c" }
```

Comment:
this is actually the `data-last` example, that would need to go on the other method please. Also, the code needs to work, and there is a missing comma here. This is why the build fails:

```
12:26:59 AM: SyntaxError: ',' expected. (5:6)
12:26:59 AM:   3 |    R.pipe(
12:26:59 AM:   4 |      { a: "d", b: "e", c: "f" }
12:26:59 AM: > 5 |      R.invert()
12:26:59 AM:     |      ^
12:26:59 AM:   6 |    ); // => { d: "a", e: "b", f: "c" }
```